src/Symfony/Component/Console/Output/ConsoleSectionOutput.php

```diff
@@ -123,7 +158,13 @@ private function popStreamContentUntilCurrentSection(int $numberOfLinesToClearFr
             }
 
             $numberOfLinesToClear += $section->lines;
-            $erasedContent[] = $section->getContent();
+            $sectionContent =  $section->getContent();
+            if ('' !== $sectionContent) {
```

Comment:
should be merged with the previous line:
`if ('' !== $sectionContent = $section->getContent()) {`